docs/docs/behaviors/tap-dance.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Tap-Dance Behavior
+sidebar_label: Tap-Dance
+---
+
+## Summary
+
+A tap-dance key invokes a different behavior (e.g. `kp`) corresponding to how many times it is pressed.
+Tap-dances are completely custom, so for every unique tap-dance key,
+a new tap-dance must be defined in your keymap's `behaviors`.
+
+Tap-dances are designed to resolve immediately when interrupted by another keypress.
+Meaning, when a keybind is pressed other than any active tap-dances,
+the tap-dance will activate according to the current value of its
+counter before the interrupting keybind is registered.
+
+### Configuration
+
+#### `tapping-term-ms`
+
+Defines the maximum elapsed time after the last tap-dance keybind press
+before a binding is selected from [`bindings`](#bindings).
+Default value is `200`ms.
+
+#### `bindings`
+
+An array of one or more keybinds. This list can include [any ZMK keycode](../codes/) and keybinds for ZMK behaviors.
+
+#### Example Usage
+
+This example configures a tap-dance that outputs the number of keypresses from 1-3.
+
+```
+#include <behaviors.dtsi>
+#include <dt-bindings/zmk/keys.h>
+
+/ {
+	behaviors {
+		td0: tap_dance_0 {
+            compatible = "zmk,behavior-tap-dance";
+            label = "TAP_DANCE_0";
+            #binding-cells = <0>;
+            tapping_term_ms = <200>;
+            bindings = <&kp N1>, <&kp N2>, <&kp N3>;
+        };
+	};
+
+	keymap {
+		compatible = "zmk,keymap";
+
+		default_layer {
+			bindings = <
+	            &td0
+			>;
+		};
+	};
+};
+```
+
+The following image describes the behavior of this particular tap-dance.
+
+![Timing Diagram](../assets/tap-dance/timing_diagram.svg)
```

Comment:
![image](https://user-images.githubusercontent.com/7876996/154879709-07c1c126-5d21-496c-a1af-c00842c0d117.png)
I think the intention here is to show an example where the tap dance key is held (blue `td` block) until `j` key is tapped. However this doesn't seem to match the diagram where the blue block cuts off before the orange block starts.

Reply:
If we read this part of the diagram with only the code in mind, the `td` block doesn't necessarily need to be held down when the `j` is tapped since the tap-dance has technically not been decided yet since the `tapping-term-ms` has not fully elapsed.
However, I definitely see your point about the confusion there since the documentation currently implies that interrupting key-bindings only trigger the tap-dance when the tap-dance key is also held down. Not sure at the moment if adding a note on the image or just putting it directly in the docs would be better. Thanks for the feedback!

Reply:
I see, it makes sense that we call it an "interrupt" also when we are waiting for a potential next tap, and not only while the key is held down (which would be the case for e.g. a hold-tap). I am also not sure how to explain it better; I can add a suggestion if I come up with one.